cascade.py

```
@@ -334,26 +355,49 @@ def single_debug(self):
+                    method = "POST"
+                )
+                with urllib.request.urlopen(req) as response:
+                    contents = response.read()
+                self.bot.send_text('Catflap is back to previous state: [' + catflap_state + '].')
+            if catflap_state == "locked_all":
+                req = urllib.request.Request(
+                    url = config.HA_LOCK_ALL_WEBHOOK,
+                    method = "POST"
+                )
+                with urllib.request.urlopen(req) as response:
+                    contents = response.read()
```

Ruff (0.11.9)

Local variable `contents` is assigned to but never used

Remove assignment to unused variable `contents`

(F841)